system-test/sanity-testcases/colo-cpu-only-quick-sanity-test.yml

```diff
@@ -6,11 +6,12 @@ steps:
       CLOUD_PROVIDER: "colo"
       TESTNET_TAG: "colo-perf-cpu-only"
       ENABLE_GPU: "false"
-      TEST_DURATION_SECONDS: 30
+      TEST_DURATION_SECONDS: 60
```

Comment:
Are the config changes besides the `TEST_TYPE` additions intentional or debug artifacts?

Reply:
Debug artifacts mostly.  I just threw the "sanity-testcases" dir in there to run short tests that exercise the functionality of the testing framework, rather than the limits of the cluster.  So I can point a buildkite job on a PR against this file rather than an expensive/long nightly-style testcase.